client/my-sites/media-library/content.jsx

```diff
@@ -45,6 +45,17 @@ class MediaLibraryContent extends React.Component {
 		filterRequiresUpgrade: PropTypes.bool,
 		search: PropTypes.string,
 		source: PropTypes.string,
+		dateRange: function( props, propName, componentName ) {
```

Comment:
> pull this custom propType out in to it's own file

👍 

> we would be checking in any component that consumes the prop, so that any prop errors point to the most relevant component.

Agreed. I'll look into this.

Reply:
I ended up duping this comment and a few others - that's what you get when reviewing in multiple tabs it seems! 😅